variables.tf

```diff
@@ -180,3 +180,22 @@ variable "kubernetes_node_count" {
   default = 2
   type    = number
 }
+
+##########################################
+# Network                                #
+##########################################
+variable "allowed_ip_ranges" {
+  description = "Map of allowed public IP addresses or CIDR ranges. Each entry in the map defines an allowed IP range along with access control settings."
+  type = map(object({
+    access                  = string
+    source_address_prefix   = string
+    priority              = number
+  }))
+  default = {
+    # block_public_rule = {
+    #   access                = "Deny"
+    #   source_address_prefix = "0.0.0.0/0"
+    #   priority              = 100
+    # }
+  }
+}
```

Comment:
What do you think about simplifying this and matching what we do in aws

https://github.com/wandb/terraform-aws-wandb/blob/main/variables.tf#L161

Reply:
This has been done.